src/modules/todos.js

```diff
@@ -0,0 +1,7 @@
+export default class Todo {
+  constructor(description) {
+    this.description = description;
+    this.completed = false;
+    this.index = null;
+  }
+}
```

Comment:
- [ ] Kindly add completed and index as a parameter inside your constructor because the current code is not pointing to them also for better code readability. 😄